meta-resin-common/recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -0,0 +1,6 @@
+[main]
+dns=default
```

Comment:
Can we have a bit more detailed git log for this commit? Some explanation maybe on why/what dns=default setting does and why we have it, then maybe also explain that we also need to ignore the docker0 interface because we don't want network managers mess with it etc.